openassessment/xblock/staff_area_mixin.py

```diff
@@ -213,6 +229,18 @@ def render_staff_grade_form(self, data, suffix=''):  # pylint: disable=W0613
                     )
                     # Add team info to context
                     submission_context['teams_enabled'] = self.teams_enabled
+                    if self.teams_enabled:
+                        user = self.get_real_user(anonymous_student_id)
+
+                        if not user:
+                            logger.error(u'{}: User lookuip for anonymous_user_id {} failed'.format(self.location, anonymous_user_id))
+                            raise ObjectDoesNotExist()
+
+                        team = self.teams_service.get_team(user, self.course_id, self.selected_teamset_id)
+
+                        submission_context['team_name'] = team.name
+                        submission_context['team_usernames'] = conv_format_list([user.username for user in team.users.all()])
```

Comment:
If you used `{{ team_usernames|join:", " }}` in the template, then you don't need this transform function (you lose the 'and', but 🤷‍♂️ )

Reply:
I know, but I generally find the conversational syntax to be more reader/user friendly.